mongo/user_storage.go

```diff
@@ -219,7 +219,7 @@ func (us *UserStorage) ImportJSON(data []byte) error {
 //data implementation
 type userData struct {
 	ID           bson.ObjectId          `bson:"_id,omitempty" json:"id,omitempty"`
-	Name         string                 `bson:"name,omitempty" json:"name,omitempty"`
+	Name         string                 `bson:"user,omitempty" json:"username,omitempty"`
```

Comment:
What is the reason?

Reply:
According to this file: [users.json](https://github.com/MadAppGang/identifo/blob/e71ab360ec4184c729346531c9888f5f222d7c62/cmd/import/users.json#L3) user's name should be parsed from the filed **username**.

Reply:
But why do you change `name` to `user`? I don't really think that `user` is an appropriate choice for the string-typed field which only holds name.